### ISSUE TITLE
Determine solution for searching curated content issues by author and EB member

### ISSUE THREAD

**Issue by bartlettroscoe** [betterscientificsoftware/bssw.io#426]
@betterscientificsoftware/editorial-board,

As discussed at the 10/3/2019 BSSw editorial board meeting, the current approach for specifying the author(s) and EB board member for a BSSw GitHub curated content issue makes it difficult to search for the list of issues that are assigned to any one person.  The current approach seems to be to list:

```
Authors: <informal-name-1>, <informal-name-2>, ...
EB member: <informal-name>
```

like #383 showing:

```
Authors: Ross B.
EB member: Mark M.
```

However, some like #348 have:

```
Author: Rinku Gupta
EB member: Pat G.
```

and #396 has:

```
Author:Paul
EB member: Mark M.
```

As shown above, even if there is just one author, it could be listed as "Authors: <name>", "Author: <name>" or "Author:<name>". 

And if there are multiple authors like:

```
Authors: Greg, Ross
```

it would be impossible to find such an issue if Ross was being searched for as an author.

And it seems that the space kills the search. For example, looking for issues containing "EB" "member" and "Ross" in [this issue query](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues?utf8=%E2%9C%93&q=EB+member+Ross+is%3Aissue+) gives the same list as for "EB member: Ross" in [this query](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues?utf8=%E2%9C%93&q=%22EB+member%3A+Ross%22+is%3Aissue+).

Therefore, even if everyone was 100% consistent with the convention, there is no way to search for the list of issues someone is an author or an EB board member.

**Comment by bartlettroscoe** [comment]
If we stick to the rule that only the EB board member is to be made the GitHub Issue assignee, then one can search for the curated content issues with:

* https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22curated+content+ideas%22+assignee%3Abartlettroscoe+

**Comment by markcmiller86** [comment]
I was about to suggest similar to what you describe...
- Indicate EB member assignment using the GitHub `Assignees` field
- Indicate Author assignment by conventional text format in *first* comment

**Comment by markcmiller86** [comment]
GitHub's issues searching has `involves:`, `mentions:`, `commenter:` and `author:` keywords for searches. All of these use a GitHub username as argument to the search.

Ignoring the potential inconsistencies of the past, would it help to use the term `content_creators` rather than `author` (which GitHub already sort of uses) as in

content_creators: Ross, Mark S., @markcmiller86

**Comment by markcmiller86** [comment]
With above, this [search](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues?utf8=✓&q=is%3Aissue+is%3Aopen+%22content_creators%3A+mark%22+) works as expected.

**Comment by markcmiller86** [comment]
You can use the above search and find this issue using
  - "content_creators: Ross"
  - "content_creators: Mark"
  - "content_creators: @markcmiller86"
  - "content_creators: mark"

So, it works for upper and lower case and it works regardless of order and presence or absence of certain entries in the names field

**Comment by markcmiller86** [comment]
Likewise, these searches fail as expected...
  - "content_creators: foo"
  - "content_creators: mark, foo"

**Comment by rinkug** [comment]
> You can use the above search and find this issue using
> 
> * "content_creators: Ross"
> * "content_creators: Mark"
> * "content_creators: @markcmiller86"
> * "content_creators: mark"
> 
> So, it works for upper and lower case and it works regardless of order and presence or absence of certain entries in the names field

@markcmiller86 : "content_creators:Mark" fails. 
But following are all passes
"content_creators : Mark", 
"content_creators:                      Mark", 
"content_creators           :        Mark     "
"content_creators :Mark" 
So, eliminating spaces completely between the "content_creators:" string and the name fails but everything else that I tested passed

**Comment by rinkug** [comment]
While we are discussing this, I wanted to let you know that I did create a PR #429  for getting the workflow document updated to the process we currently use. Hence there is a line in that PR to indicate author but "Author : Foo". I would like this PR to be anyways merged into master.

Once we finalize this issue, we can always update the workflow.

**Comment by bartlettroscoe** [comment]
FYI: This should be fixed this in commit 2522cdd in PR #429.

We can close this issue #426 that once that PR #429 is merged.

**Comment by rinkug** [comment]
This should have been closed long ago since the fix is in
